tests/serializers/models/base.py

```diff
@@ -52,6 +52,21 @@ def __str__(self):
         return self.name
 
 
+class GhostWriterManager(models.Manager):
+
+    def get_by_natural_key(self, *args, **kwargs):
+        raise NotImplementedError("Don't get by natural key")
```

Comment:
It should be enough to raise `NotImplementedError` in `natural_key()`, so I would return a base implementation here, e.g.
```suggestion
        return self.get(*args, **kwargs)
```